swagger-spec/schema_responses/definition.yaml

```diff
@@ -0,0 +1,119 @@
+type: object
+title: Schema Responses
+required:
+  - type
+  - attributes
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: 'The unique identifier of the Schema Response.'
+  type:
+    type: string
+    readOnly: false
+    description: 'The type identifier of the Schema Response (`schema-response`).'
+  attributes:
+    type: object
+    title: Attributes
+    readOnly: false
+    description: 'The properties of the Schema Response.'
+    properties:
+      revision_justification:
+        type: string
+        readOnly: false
+        description: 'A string representing the reason the Schema Response was updated.'
+      date_created:
+        type: integer
+        readOnly: false
+        description: 'The date the Schema Response was created'
+      date_submitted:
+        type: integer
+        readOnly: false
+        description: 'The date the Schema Response was submitted for approval.'
+      date_modified:
+        type: integer
+        readOnly: false
+        description: 'The date the Schema Response was most recently changed.'
+      revision_responses:
+        type: object
+        items:
+          type: object
+          description: 'Revision responses use a custom key value pair where the key is the block key for that the
+          response. For example in the registration response `{"q1": "Answer 1"}` the block key is `q1`.'
+        description: 'A dictionary object representing responses to to the revision where the key is the block key for
+        that the response. For example in the registration response `{"q1": "Answer 1"}` the block key is `q1`..'
+      reviews_state:
+        type: string
+        readOnly: false
+        enum:
+          - initial
+          - in_progress
+          - approved
+          - pending_moderation
+          - rejected
+          - rejected_moderation
```

Comment:
I don't think these two are valid states. Wouldn't `rejected` and `rejected_moderation` just be `in_progress`?